### PR TITLE
Clean up errors

### PR DESCRIPTION
**Pull request by rafapaezbas**

### CODE REVIEW

File: gui/gui.js [hunk 1]
```diff
@@ -837,7 +845,7 @@ class GuiCtrl {
   }
 
   async send (...args) {
-    if (this.closed) throw Error(`Cannot send to closed ${this[kCtrl]}`)
+    if (this.closed) throw CANNOT_SEND(`Cannot send to closed ${this[kCtrl]}`)
```
Comment (mafintosh) [inline gui/gui.js:848]: this one should be INSTANCE_CLOSED or similar

File: gui/gui.js [hunk 2]
```diff
@@ -877,7 +885,7 @@ class GuiCtrl {
   }
 
   async getMediaSourceId () {
-    if (this.closed) throw Error(`Cannot get media source id if the ${this[kCtrl]} is closed`)
+    if (this.closed) throw CANNOT_GET_MEDIA_SOURCE_ID(`Cannot get media source id if the ${this[kCtrl]} is closed`)
```
Comment (mafintosh) [inline gui/gui.js:888]: same as above

File: gui/gui.js (hunk 1 above)
Comment (rafapaezbas) [reply gui/gui.js]: done

File: gui/gui.js (hunk 2 above)
Comment (rafapaezbas) [reply gui/gui.js]: done

File: cmd/iface.js [hunk 3]
```diff
@@ -60,19 +60,20 @@ function indicator (value, type = 'success') {
...
       if (json) {
         print(JSON.stringify({ cmd, tag, data }))
         continue
       }
-      if (tag === 'error') {
-        const err = new Error(data.message)
-        err.sidecarCode = data.code
-        err.sidecarStack = data.stack
-        throw err
+      let result = null
```
Comment (davidmarkclements) [inline cmd/iface.js:70]: have you checked that this change doesn't cause breakage for other command outputs? it definitely will if any of them push objects with tag: 'error' 

what's the reason for this change?

Comment (rafapaezbas) [reply cmd/iface.js:70]: sorry, that was unintended, let me revert
